### DISCUSSION TITLE
Security / Linux updates

### DISCUSSION THREAD

**Discussion by vvevans** [WLAN-Pi/feedback#4] [General Inquiries] [answered]
What is the rule of thumb for applying patches, updates, etc. to Linux on the WLAN Pi?

When I log in, I see that there are updates available, however I have avoided performing "apt get upgrade". Should we generally just wait for the next wlanpi image?

Thank you ahead of time! Love this tool.

**Comment by jolla** [comment, marked as answer]
No issues to be aware of, update away.
